src/patterns.md

```diff
@@ -0,0 +1,646 @@
+# Patterns
+
+> **<sup>Syntax</sup>**\
+> _Pattern_ :\
+> &nbsp;&nbsp; &nbsp;&nbsp; [_LiteralPattern_]\
+> &nbsp;&nbsp; | [_IdentifierPattern_]\
+> &nbsp;&nbsp; | [_WildcardPattern_]\
+> &nbsp;&nbsp; | [_RangePattern_]\
+> &nbsp;&nbsp; | [_ReferencePattern_]\
+> &nbsp;&nbsp; | [_StructPattern_]\
+> &nbsp;&nbsp; | [_TupleStructPattern_]\
+> &nbsp;&nbsp; | [_TuplePattern_]\
+> &nbsp;&nbsp; | [_SlicePattern_]\
+> &nbsp;&nbsp; | [_PathPattern_]
+
+Patterns in Rust are used to match values against structures and to,
+optionally, bind variables to values inside these structures. They are also
+used in variable declarations and function/closure parameters, though in these
```

Comment:
General style right now is to not use `x/y`. Could rewrite as "parameters for functions and closures".

Reply:
The clause after the comma could be its own sentence, or maybe even removed. If not removed, "though in these cases most of the time" -> "though in most of these cases,".